training/src/main/java/wolox/training/models/Client.java

```diff
@@ -48,6 +47,7 @@ public LocalDate getBirthdate() {
     }
 
     public void setBirthdate(LocalDate birthdate) {
+        Preconditions.checkNotNull(birthdate);
```

Comment:
You can directly assign the return of a checkNotNull ;)

Reply:
Asi ? 
Preconditions.checkArgument(title != null and !title.isEmpty()).set(title)